packages/models-library/src/models_library/services.py

```diff
@@ -32,7 +33,7 @@
 )
 
 # CONSTANTS -------------------------------------------
-
+# NOTE: move to _constants.py: SEE https://github.com/ITISFoundation/osparc-simcore/issues/3486
```

Comment:
is this a disguised TODO? if you created an issue, it probably does not need to be *as well* in here no?

Reply:
It is indeed a TODO. I like to keep the link in the code as a reminder but i can also remove it if the team finds it redundant